TransferMethod/Tests/AddTransferMethodPresenterTests.swift

```diff
@@ -78,6 +96,22 @@ class AddTransferMethodPresenterTests: XCTestCase {
         XCTAssertFalse(mockView.isShowErrorPerformed, "The showError should not be performed")
         XCTAssertTrue(mockView.isShowConfirmationPerformed, "The showConfirmation should be performed")
         XCTAssertTrue(mockView.isNotificationSent, "The notification should be sent")
+
+        XCTAssert(mockHyperwalletInsights.trackClick, "HyperwalletInsights.trackClick should be called")
+        XCTAssert(mockHyperwalletInsights.trackImpression, "HyperwalletInsights.trackImpression should be called")
+        XCTAssert(mockHyperwalletInsights.pageName == "transfer-method:add:collect-transfer-method-information",
+                  "Page name should be transfer-method:add:collect-transfer-method-information")
+        XCTAssert(mockHyperwalletInsights.pageGroup == "transfer-method",
+                  "Page group should be transfer-method")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.country] != nil),
+                  "Params should have country")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.currency] != nil),
+                  "Params should have currency")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.profileType] != nil),
+                  "Params should have profile type")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.transferMethodType] != nil),
+                  "Params should have transfer method type")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.goal] != nil), "Params should have goal")
```

Comment:
Use XCTAssertEqual instead of checking ==
Use XCTAssertTrue instead of XCTAssert
Use XCTAssertNotNil instead of !=nil

Reply:
It has been updated based #165 PR changes.